### PR TITLE
chore: fix windows install

### PR DESCRIPTION
**Pull request by hansieodendaal**
Description
---
Fixed Windows Install

Motivation and Context
---
Windows installation was not working properly

How Has This Been Tested?
---
System level install file generation, installed using install file, run all installed components

What process can a PR reviewer use to test or verify this change?
---
System level install file generation, installed using install file, run all installed components

Breaking Changes
---

- [x] None
- [ ] Requires data directory on base node to be deleted
- [ ] Requires hard fork
- [ ] Other - Please specify

### CODE REVIEW

**Review by leet4tari — COMMENTED** [review]
LGTM

File: base_layer/mmr/src/sparse_merkle_tree/mod.rs [hunk 1]
```diff
@@ -25,12 +25,8 @@
...
 //! fourth bit. This results in a SMT that looks like this:
 //!
-//! .           ┌──────┐
-//!       ┌─────┤ root ├─────┐
-//!       │     └──────┘     │
-//!      ┌┴┐                ┌┴┐1
-//!   ┌──┤ ├──┐          ┌──┤ ├───┐
-//!   │  └─┘  │          │  └─┘   │
+//! . ┌──────┐ ┌─────┤ root ├─────┐ │     └──────┘     │ ┌┴┐                ┌┴┐1 ┌──┤ ├──┐          ┌──┤ ├───┐ │  └─┘  │
+//!   │  └─┘   │
```
Comment (SWvheerden) [inline base_layer/mmr/src/sparse_merkle_tree/mod.rs]: this is wrong?

**Review by SWvheerden — COMMENTED** [review]
just a nit

File: base_layer/mmr/src/sparse_merkle_tree/mod.rs (hunk 1 above)
Comment (hansieodendaal) [reply base_layer/mmr/src/sparse_merkle_tree/mod.rs]: Yip, it is the `.` in `//! .` that made the issue with cargo format.

**Review by SWvheerden — APPROVED** [review]

**Pull request merged**